eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -73,7 +88,7 @@ jobs:
         inputs:
           mavenPomFile: sdk/${{parameters.ServiceDirectory}}/pom.service.xml
           goals: 'package'
-          options: '$(DefaultOptions) "-DpackageOutputDirectory=$(Build.ArtifactStagingDirectory)" -DskipTests -Dinclude-template -Dgenerate-overview' # We include template-module so we ensure it always builds in CI
```

Comment:
Did you intend to remove the "include-template" as part of this PR?

Reply:
Yeah ... I think it is redundant now that we have a separate pipeline for it (which we frequently use for testing).